### PR TITLE
Bug 1272358 - When creating a big bundle via UI the wizard shows erro…

### PR DESCRIPTION
**Pull request by josejulio**
…rs if user clicks Next button multiple times

Added a flag to know if the token processing(User doesn't have permission to add withou Bundle Group, so a token processing is needed) is and show a message, similar to the one when shows when click next while uploading a large bundle.

### CODE REVIEW

**Comment by rhqci** [comment]
Can one of the admins verify this patch?

**Comment by pilhuhn** [comment]
@jshaughn Can you have a look at this please?

File: modules/enterprise/gui/coregui/src/main/java/org/rhq/coregui/client/bundle/create/BundleGroupsStep.java [hunk 1]
```diff
@@ -62,11 +62,15 @@
     // a liberal timeout here...
     private static final int BUNDLE_CREATE_INITIAL_VERSION_TIMEOUT = 10 * 60 * 1000;
 
+    private static final int PROCESSING_IDLE = 0;
```
Comment (burmanm) [inline modules/enterprise/gui/coregui/src/main/java/org/rhq/coregui/client/bundle/create/BundleGroupsStep.java]: Don't use ints to emulate enums

Comment (josejulio) [inline modules/enterprise/gui/coregui/src/main/java/org/rhq/coregui/client/bundle/create/BundleGroupsStep.java]: Ok, will change that to actual enums.

**Comment by jshaughn** [comment]
Looks OK to me

**Pull request merged**